views/js/qtiCreator/helper/qtiElements.js

```diff
@@ -299,158 +299,164 @@ define(['jquery', 'lodash', 'i18n'], function($, _, __){
-                tags:['Common Interactions', 'text']
+                tags:[tagTitles.commonInteractions, 'text']
             },
             gapMatchInteraction : {
                 label : __('Gap Match Interaction'),
                 icon : 'icon-gap-match',
                 description : __(' Fill in the gaps in a text from a set of choices.'),
                 short : __('Gap Match'),
                 qtiClass : 'gapMatchInteraction',
-                tags:['Common Interactions', 'text', 'association']
+                tags:[tagTitles.commonInteractions, 'text', 'association']
             },
             sliderInteraction : {
                 label : __('Slider Interaction'),
                 icon : 'icon-slider',
                 description :__('Select a value within a numerical range.'),
                 short : __('Slider'),
                 qtiClass : 'sliderInteraction',
-                tags:['Common Interactions', 'special']
+                tags:[tagTitles.commonInteractions, 'special']
             },
             extendedTextInteraction : {
                 label : __('Extended Text Interaction'),
                 icon : 'icon-extended-text',
                 description : __('Collect open-ended information in one or more text area(s) (strings or numeric values).'),
                 short : __('Extended Text'),
                 qtiClass : 'extendedTextInteraction',
-                tags:['Common Interactions', 'text']
+                tags:[tagTitles.commonInteractions, 'text']
             },
             uploadInteraction : {
                 label : __('File Upload Interaction'),
                 icon : 'icon-upload',
                 description : __('Upload a file (e.g. document, picture...) as a response.'),
                 short : __('File Upload'),
                 qtiClass : 'uploadInteraction',
-                tags:['Common Interactions', 'special']
+                tags:[tagTitles.commonInteractions, 'special']
             },
             mediaInteraction : {
                 label : __('Media Interaction'),
                 icon : 'icon-media',
                 description : __('Control the playing parameters (auto-start, loop) of a video or audio file and report the number of time it has been played.'),
                 short : __('Media'),
                 qtiClass : 'mediaInteraction',
-                tags:['Common Interactions', 'media']
+                tags:[tagTitles.commonInteractions, 'media']
             },
             _container : {
                 label : __('Text Block'),
                 icon : 'icon-font',
                 description : __('Block contains the content (stimulus) of the item such as text or image. It is also required for Inline Interactions.'),
                 short : __('Block'),
                 qtiClass : '_container',
-                tags:['Inline Interactions', 'text']
+                tags:[__('Inline Interactions'), 'text']
```

Comment:
As other strings already moved to object with lines, it seems missing.

Reply:
Nice catch!